src/zh/baozimhorg/src/eu/kanade/tachiyomi/extension/zh/baozimhorg/Dto.kt

```diff
@@ -40,13 +40,33 @@ class AttributesDto(
     fun toSChapter(mangaSlug: String, mangaId: String, chapterId: String) = SChapter.create().apply {
         url = "$mangaSlug/$slug#$mangaId/$chapterId"
         name = title
-        date_upload = dateFormat.parse(updatedAt)!!.time
+        date_upload = parseDate(updatedAt)
     }
 }
 
 // Static field, no need for lazy
-private val dateFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'", Locale.US).apply {
-    timeZone = TimeZone.getTimeZone("UTC")
+// private val dateFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'", Locale.US).apply {
+//    timeZone = TimeZone.getTimeZone("UTC")
+// }
+
```

Comment:
just remove it if it is no longer needed

Reply:
My bad, forget to remove old code